src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -2029,7 +2147,12 @@ private TokenFlags ScanStringExpandable(StringBuilder sb, StringBuilder formatSb
                     if (c1 != 0)
                     {
                         SkipChar();
-                        c = Backtick(c1);
+                        string str = Backtick(c1);
+                        if (IsSurrogatePair(str, ref c, sb, formatSb))
+                        {
+                            // Skip further scanning when str is surrogate pair and is outside the basic multilingual plane
+                            continue;
```

Comment:
This comment isn't right - no scanning is skipped - it should say the character has been processed and appended to the string builders.